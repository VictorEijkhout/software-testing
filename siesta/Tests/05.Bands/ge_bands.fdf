%include ../base_ge.fdf
SystemLabel       ge_bands

