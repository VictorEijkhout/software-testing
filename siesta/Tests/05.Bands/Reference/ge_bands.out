Siesta Version  : 5.0.0-alpha-123-gfb8fd7d02
Architecture    : ----
Compiler version: GNU-11.4.0
Compiler flags  : -fallow-argument-mismatch;-O3 -march=native
PP flags        : ----
Libraries       : ----
Parallelisations: none
GEMM3M support
NetCDF support
NetCDF-4 support
Lua support

Runtime information:
* Directory : /home/fnpedron/Siesta/siesta-dev/Tests/05.Bands/work/ge_bands
* Running in serial mode.
>> Start of run:  28-SEP-2023  15:36:02

                           *********************** 
                           *  WELCOME TO SIESTA  * 
                           *********************** 

reinit: Reading from standard input
reinit: Dumping input in INPUT_TMP.56517
************************** Dump of input data file ****************************
%include ../../basejob.fdf
SystemLabel       ge_bands
************************** End of input data file *****************************

reinit: -----------------------------------------------------------------------
reinit: System Name: bulk Ge with spin-orbit. Test bands
reinit: -----------------------------------------------------------------------
reinit: System Label: ge_bands
reinit: -----------------------------------------------------------------------

initatom: Reading input for the pseudopotentials and atomic orbitals ----------
Species number:   1 Atomic number:   32 Label: Ge


---- Processing specs for species: Ge
Ground state valence configuration:   4s02  4p02
Reading pseudopotential information in formatted form from:
  /home/fnpedron/Siesta/siesta-dev/Tests/Pseudos/Ge.psf

---- Pseudopotential check for Ge

Pseudopotential generated from a fully relativistic atomic calculation
There are spin-orbit semi-local pseudopotentials available

Pseudized shells:
4s( 2.00) rc: 2.25
4p( 2.00) rc: 2.99
4d( 0.00) rc: 2.48
4f( 0.00) rc: 2.48
Valence configuration for ps generation: (assumed as above)
For Ge, standard SIESTA heuristics set lmxkb to 3
 (one more than the basis l, including polarization orbitals).
Use PS.lmax or PS.KBprojectors blocks to override.

<basis_specs>
===============================================================================
Ge                   Z=  32    Mass=  72.630        Charge= 0.17977+309
Lmxo=2 Lmxkb= 3    BasisType=split      Semic=F
L=0  Nsemic=0  Cnfigmx=4
          i=1  nzeta=1  polorb=0  (4s)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=1  Nsemic=0  Cnfigmx=4
          i=1  nzeta=1  polorb=1  (4p)  (to be polarized perturbatively)
            splnorm:   0.15000    
               vcte:    40.000    
               rinn:  -0.90000    
               qcoe:    0.0000    
               qyuk:    0.0000    
               qwid:   0.10000E-01
                rcs:    0.0000    
            lambdas:    1.0000    
L=2  Nsemic=0  Cnfigmx=4
          i=1  nzeta=0  polorb=0  (4d)  (perturbative polarization orbital) (from 4p)
-------------------------------------------------------------------------------
L=0  Nkbl=1  erefs: 0.17977+309
L=1  Nkbl=1  erefs: 0.17977+309
L=2  Nkbl=1  erefs: 0.17977+309
L=3  Nkbl=1  erefs: 0.17977+309
===============================================================================
</basis_specs>

Using PAO.SplitTailNorm
atom: Called for Ge                    (Z =  32)
Maximum radius (at nrval) set to   50.00000 8518

read_vps: Pseudopotential generation method:
read_vps: ATM3      Troullier-Martins                       
Valence charge in pseudo generation:    4.00000

read_vps: Pseudopotential includes a core correction:
read_vps: Pseudo-core for xc-correction

xc_check: Exchange-correlation functional:
xc_check: Ceperley-Alder
V l=0 = -2*Zval/r beyond r=  3.2864
V l=1 = -2*Zval/r beyond r=  3.2864
V l=2 = -2*Zval/r beyond r=  3.2864
V l=3 = -2*Zval/r beyond r=  3.2864
All V_l potentials equal beyond r=  2.9460
This should be close to max(r_c) in ps generation
All pots = -2*Zval/r beyond r=  3.2864
Using small-core scheme (local charge) for Vlocal

VLOCAL_FROM_CHLOCAL: 99.0% of the norm of Vloc inside      5.002 Ry
VLOCAL_FROM_CHLOCAL: 99.9% of the norm of Vloc inside     11.399 Ry
atom: Maximum radius for chlocal:    3.47626
atom: Maximum radius for r*vlocal+2*Zval:    3.22436

KBgen: Kleinman-Bylander projectors: 
GHOST: No ghost state for L =  0
   l= 0   rc=  3.410647   el= -0.876405   Ekb=  4.925561   kbcos=  0.219549
GHOST: No ghost state for L =  1
   l= 1   rc=  3.607669   el= -0.297940   Ekb=  1.243727   kbcos=  0.209275
GHOST: No ghost state for L =  2
   l= 2   rc=  3.714117   el=  0.013077   Ekb=  2.404306   kbcos=  0.039174
GHOST: No ghost state for L =  3
   l= 3   rc=  3.885530   el=  0.019532   Ekb= -0.812032   kbcos= -0.018235

KBgen: Total number of Kleinman-Bylander projectors:  16
atom: -------------------------------------------------------------------------

atom: SANKEY-TYPE ORBITALS:

SPLIT: Orbitals with angular momentum L= 0

SPLIT: Basis orbitals for state 4s

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.022050 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    4.781434
                 energy =   -0.851598
                kinetic =    0.603123
    potential(screened) =   -1.454720
       potential(ionic) =   -3.810395

SPLIT: Orbitals with angular momentum L= 1

SPLIT: Basis orbitals for state 4p

SPLIT: PAO cut-off radius determined from an
SPLIT: energy shift=  0.022050 Ry
Split based on tail norm

   izeta = 1
                 lambda =    1.000000
                     rc =    6.179348
                 energy =   -0.273070
                kinetic =    0.862278
    potential(screened) =   -1.135348
       potential(ionic) =   -3.309909

POLgen: Perturbative polarization orbital with L=  2

POLgen: Polarization orbital for state 4p
Split based on tail norm

   izeta = 1
                     rc =    6.179348
                 energy =    0.484917
                kinetic =    1.412243
    potential(screened) =   -0.927326
       potential(ionic) =   -2.979922
atom: Total number of Sankey-type orbitals:  9

atm_pop: Valence configuration (for local Pseudopot. screening):
 4s( 2.00)                                                            
 4p( 2.00)                                                            
 4d( 0.00)                                                            
Vna: chval, zval:    4.00000   4.00000

Vna:  Cut-off radius for the neutral-atom potential:   6.179348
comcore: Pseudo-core radius Rcore=  4.435222

atom: _________________________________________________________________________

prinput: Basis input 

PAO.BasisType split     

%block ChemicalSpeciesLabel
    1   32 Ge                      # Species index, atomic number, species label
%endblock ChemicalSpeciesLabel

%block PAO.Basis                 # Define Basis set
# WARNING: This information might be incomplete!
Ge                    2                    # Species label, number of l-shells
 n=4   0   1                         # n, l, Nzeta 
   4.781   
   1.000   
 n=4   1   1 P   1                   # n, l, Nzeta, Polarization, NzetaPol
   6.179   
   1.000   
%endblock PAO.Basis

prinput: ----------------------------------------------------------------------

Dumping basis to NetCDF file Ge.ion.nc
coor:   Atomic-coordinates input format  =     Fractional

siesta: Atomic coordinates (Bohr) and species
siesta:      0.00000   0.00000   0.00000  1        1
siesta:      2.67396   2.67396   2.67396  1        2

siesta: System type = bulk      

initatomlists: Number of atoms, orbitals, and projectors:      2    18    32

siesta: ******************** Simulation parameters ****************************
siesta:
siesta: The following are some of the parameters of the simulation.
siesta: A complete list of the parameters used, including default values,
siesta: can be found in file out.fdf
siesta:
redata: Spin configuration                          = none
redata: Number of spin components                   = 1
redata: Time-Reversal Symmetry                      = T
redata: Spin spiral                                 = F
redata: Long output                                 =   F
redata: Number of Atomic Species                    =        1
redata: Charge density info will appear in .RHO file
redata: Write Mulliken Pop.                         = NO
redata: Matel table size (NRTAB)                    =     1024
redata: Mesh Cutoff                                 =   200.0000 Ry
redata: Net charge of the system                    =     0.0000 |e|
redata: Min. number of SCF Iter                     =        0
redata: Max. number of SCF Iter                     =       50
redata: SCF convergence failure will abort job
redata: SCF mix quantity                            = Hamiltonian
redata: Mix DM or H after convergence               =   F
redata: Recompute H after scf cycle                 =   F
redata: Mix DM in first SCF step                    =   T
redata: Write Pulay info on disk                    =   F
redata: New DM Occupancy tolerance                  = 0.000000000001
redata: No kicks to SCF
redata: DM Mixing Weight for Kicks                  =     0.5000
redata: Require Harris convergence for SCF          =   F
redata: Harris energy tolerance for SCF             =     0.000100 eV
redata: Require DM convergence for SCF              =   T
redata: DM tolerance for SCF                        =     0.000100
redata: Require EDM convergence for SCF             =   F
redata: EDM tolerance for SCF                       =     0.001000 eV
redata: Require H convergence for SCF               =   T
redata: Hamiltonian tolerance for SCF               =     0.001000 eV
redata: Require (free) Energy convergence for SCF   =   F
redata: (free) Energy tolerance for SCF             =     0.000100 eV
redata: Using DFT-D3 dispersion                     =   F
redata: Using Saved Data (generic)                  =   F
redata: Use continuation files for DM               =   F
redata: Neglect nonoverlap interactions             =   F
redata: Method of Calculation                       = Diagonalization
redata: Electronic Temperature                      =   290.1130 K
redata: Fix the spin of the system                  =   F
redata: Max. number of TDED Iter                    =        1
redata: Number of TDED substeps                     =        3
redata: Dynamics option                             = Single-point calculation
mix.SCF: Pulay mixing                            = Pulay
mix.SCF:    Variant                              = stable
mix.SCF:    History steps                        = 3
mix.SCF:    Linear mixing weight                 =     0.300000
mix.SCF:    Mixing weight                        =     0.300000
mix.SCF:    SVD condition                        = 0.1000E-07
redata: Save all siesta data in one NC              =   F
redata: ***********************************************************************

%block SCF.Mixers
  Pulay
%endblock SCF.Mixers

%block SCF.Mixer.Pulay
  # Mixing method
  method pulay
  variant stable

  # Mixing options
  weight 0.3000
  weight.linear 0.3000
  history 3
%endblock SCF.Mixer.Pulay

DM_history_depth set to one: no extrapolation allowed by default for geometry relaxation
Size of DM history Fstack: 1
Total number of electrons:     8.000000
Total ionic charge:     8.000000
 Kpoints in:           32 . Kpoints trimmed:           32

siesta: k-grid: Number of k-points = 32
siesta: k-points from cutoff
siesta: k-cutoff (effective) =     8.004 Ang
siesta: k-point supercell and displacements
siesta: k-grid:     0     4     0      0.500
siesta: k-grid:     0     0     4      0.500
siesta: k-grid:     4     0     0      0.500

diag: Algorithm                                     = D&C
diag: Used triangular part                          = Lower
diag: Absolute tolerance                            =  0.100E-15
diag: Orthogonalization factor                      =  0.100E-05
diag: Memory factor                                 =  1.0000
Using LatticeConstant from fdf file for BandLinesScale:   10.695850 Bohr
Beware any cell changes by the end of the run
Using LatticeConstant from fdf file for BandLinesScale:   10.695850 Bohr
Beware any cell changes by the end of the run

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   2250   4000


ts: **************************************************************
ts: Save H and S matrices                           =    F
ts: Save DM and EDM matrices                        =    F
ts: Only save the overlap matrix S                  =    F
ts: **************************************************************

************************ Begin: TS CHECKS AND WARNINGS ************************
************************ End: TS CHECKS AND WARNINGS **************************


                     ====================================
                        Single-point calculation
                     ====================================

superc: Internal auxiliary supercell:     5 x     5 x     5  =     125
superc: Number of atoms, orbitals, and projectors:    250   2250   4000

outcell: Unit cell vectors (Ang):
        2.830000    2.830000    0.000000
        0.000000    2.830000    2.830000
        2.830000    0.000000    2.830000

outcell: Cell vector modules (Ang)   :    4.002224    4.002224    4.002224
outcell: Cell angles (23,13,12) (deg):     60.0000     60.0000     60.0000
outcell: Cell volume (Ang**3)        :     45.3304
<dSpData1D:S at geom step 0
  <sparsity:sparsity for geom step 0
    nrows_g=18 nrows=18 sparsity=70.7346 nnzs=22918, refcount: 7>
  <dData1D:(new from dSpData1D) n=22918, refcount: 1>
refcount: 1>
new_DM -- step:     1
Initializing Density Matrix...
DM filled with atomic data:
<dSpData2D:DM initialized from atoms
  <sparsity:sparsity for geom step 0
    nrows_g=18 nrows=18 sparsity=70.7346 nnzs=22918, refcount: 8>
  <dData2D:DM n=22918 m=1, refcount: 1>
refcount: 1>
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      38     249

InitMesh: MESH = 30 x 30 x 30 = 27000
InitMesh: Mesh cutoff (required, used) =   200.000   232.934 Ry
New grid distribution [1]: sub = 2

stepf: Fermi-Dirac step function

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -73.607561
siesta: Eions   =       264.217052
siesta: Ena     =        27.541614
siesta: Ekin    =        73.655502
siesta: Enl     =        14.688330
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         5.294414
siesta: DUscf   =         0.341189
siesta: DUext   =         0.000000
siesta: Ex      =      -133.969380
siesta: Ec      =       -20.232219
siesta: Exc     =      -154.201599
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -298.593044
siesta: Etot    =      -296.897603
siesta: FreeEng =      -296.897603

        iscf     Eharris(eV)        E_KS(eV)     FreeEng(eV)     dDmax    Ef(eV) dHmax(eV)
   scf:    1     -298.593044     -296.897603     -296.897603  0.906943 -3.185491  0.520517
timer: Routine,Calls,Time,% = IterSCF        1       0.068  13.51
   scf:    2     -296.913159     -296.905478     -296.905478  0.012677 -3.305782  0.344682
   scf:    3     -296.916893     -296.911532     -296.911532  0.025241 -3.536266  0.004599
   scf:    4     -296.911557     -296.911546     -296.911546  0.001021 -3.536105  0.000664
   scf:    5     -296.911546     -296.911546     -296.911546  0.000088 -3.536045  0.000674

SCF Convergence by DM+H criterion
max |DM_out - DM_in|         :     0.0000880815
max |H_out - H_in|      (eV) :     0.0006742913
SCF cycle converged after 5 iterations

Using DM_out to compute the final energy and forces
No. of atoms with KB's overlaping orbs in proc 0. Max # of overlaps:      38     249

siesta: E_KS(eV) =             -296.9115

siesta: E_KS - E_eggbox =      -296.9115

siesta: Atomic forces (eV/Ang):
----------------------------------------
   Tot   -0.000119   -0.000119   -0.000119
----------------------------------------
   Max    0.022894
   Res    0.022834    sqrt( Sum f_i^2 / 3N )
----------------------------------------
   Max    0.022894    constrained

Stress tensor Voigt[x,y,z,yz,xz,xy] (kbar):       -6.29       -6.29       -6.29        9.33        9.33        9.33
(Free)E + p*V (eV/cell)     -296.7334
Target enthalpy (eV/cell)     -296.9115
Computing bands...

siesta: Program's energy decomposition (eV):
siesta: Ebs     =       -73.274514
siesta: Eions   =       264.217052
siesta: Ena     =        27.541614
siesta: Ekin    =        74.101546
siesta: Enl     =        14.878387
siesta: Eso     =         0.000000
siesta: Edftu   =         0.000000
siesta: DEna    =         4.903094
siesta: DUscf   =         0.305168
siesta: DUext   =         0.000000
siesta: Ex      =      -134.178879
siesta: Ec      =       -20.245425
siesta: Exc     =      -154.424304
siesta: EbV     =         0.000000
siesta: eta*DQ  =         0.000000
siesta: Emadel  =         0.000000
siesta: Emeta   =         0.000000
siesta: Emolmec =         0.000000
siesta: Ekinion =         0.000000
siesta: Eharris =      -296.911546
siesta: Etot    =      -296.911546
siesta: FreeEng =      -296.911546

siesta: Final energy (eV):
siesta:  Band Struct. =     -73.274514
siesta:       Kinetic =      74.101546
siesta:       Hartree =      14.324214
siesta:         Edftu =       0.000000
siesta:       Eso     =       0.000000
siesta:    Ext. field =       0.000000
siesta:         Exch. =    -134.178879
siesta:         Corr. =     -20.245425
siesta:     Bulk bias =       0.000000
siesta:   Exch.-corr. =    -154.424304
siesta:  Ion-electron =     -82.914584
siesta:       Ion-ion =    -147.998418
siesta:       Ekinion =       0.000000
siesta: D3 dispersion =       0.000000
siesta:         Total =    -296.911546
siesta:         Fermi =      -3.536045

siesta: Stress tensor (static) (eV/Ang**3):
siesta:    -0.003929    0.005826    0.005826
siesta:     0.005826   -0.003929    0.005826
siesta:     0.005826    0.005826   -0.003929

siesta: Cell volume =         45.330374 Ang**3

siesta: Pressure (static):
siesta:                Solid            Molecule  Units
siesta:           0.00004279          0.00003503  Ry/Bohr**3
siesta:           0.00392893          0.00321616  eV/Ang**3
siesta:           6.29484749          5.15285297  kBar
(Free)E+ p_basis*V_orbitals  =        -296.010838
(Free)Eharris+ p_basis*V_orbitals  =        -296.010839

cite: Please see "ge_bands.bib" for an exhaustive BiBTeX file.
cite: Please clearly indicate Siesta version in published work: 5.0.0-alpha-123-gfb8fd7d02
cite: This calculation has made use of the following articles
cite: which are encouraged to be cited in a published work.
        Primary SIESTA paper
          DOI: www.doi.org/10.1088/0953-8984/14/11/302


timer: Elapsed wall time (sec) =       1.017
timer: CPU execution times (sec):

Routine            Calls   Time/call    Tot.time        %
siesta                 1       1.017       1.017   100.00
Setup                  1       0.040       0.040     3.98
bands                  3       0.007       0.022     2.13
KSV_init               1       0.000       0.000     0.00
IterGeom               1       0.954       0.954    93.83
geom_init              1       0.394       0.394    38.73
state_init             1       0.035       0.035     3.48
hsparse                1       0.003       0.003     0.26
overlap                1       0.027       0.027     2.64
Setup_H0               1       0.314       0.314    30.90
naefs                  2       0.000       0.000     0.03
MolMec                 2       0.000       0.000     0.00
kinefsm                2       0.013       0.027     2.63
nlefsm                 2       0.125       0.251    24.66
DHSCF_Init             1       0.104       0.104    10.27
DHSCF1                 1       0.009       0.009     0.93
INITMESH               1       0.000       0.000     0.00
DHSCF2                 1       0.095       0.095     9.34
REMESH                 1       0.011       0.011     1.11
REORD                 29       0.000       0.001     0.07
PHION                  1       0.071       0.071     7.02
COMM_BSC              31       0.000       0.001     0.05
POISON                 8       0.000       0.004     0.35
fft                   16       0.000       0.002     0.22
setup_H                6       0.044       0.262    25.82
DHSCF                  7       0.059       0.412    40.51
DHSCF3                 7       0.044       0.305    29.96
rhoofd                 7       0.024       0.167    16.41
XC                     7       0.002       0.014     1.36
GXC-CellXC             7       0.002       0.014     1.34
gridxc@cellXC          7       0.002       0.014     1.33
vmat                   7       0.017       0.119    11.69
IterSCF                5       0.068       0.340    33.44
compute_dm             5       0.024       0.120    11.79
diagon                 5       0.024       0.120    11.77
c-eigval             256       0.000       0.060     5.86
c-buildHS            256       0.000       0.054     5.26
cdiag                416       0.000       0.013     1.31
cdiag1               416       0.000       0.001     0.10
cdiag2               416       0.000       0.002     0.20
cdiag3               416       0.000       0.009     0.86
c-eigvec             160       0.000       0.042     4.11
cdiag4               160       0.000       0.000     0.02
c-buildD             160       0.000       0.039     3.83
MIXER                  4       0.000       0.001     0.06
PostSCF                1       0.220       0.220    21.63
DHSCF4                 1       0.107       0.107    10.55
dfscf                  1       0.094       0.094     9.21
overfsm                1       0.002       0.002     0.19
state_analysis         1       0.000       0.000     0.01
siesta_move            1       0.000       0.000     0.00
Analysis               1       0.022       0.022     2.16
optical                1       0.000       0.000     0.00
  

>> End of run:  28-SEP-2023  15:36:03
Job completed
